site/book-theme/header.hbs

```diff
@@ -0,0 +1,173 @@
+<style>
+.main-header {
+  /* Keep this high-z to stop fwd/prev nav from overlapping it. */
+  z-index: 102;
+  position: sticky;
+  background-image: linear-gradient(90deg, #4BC0C8 0%, #C779D0 50%, #FEAC5E 100%);
+  top: 0;
+}
+
+.main-header__container {
+  margin: 0 auto;
+  margin-inline: 2em;
+  display: flex;
+  flex-wrap: nowrap;
+  justify-content: flex-start;
+  align-items: center;
+}
+
+.logo {
+  font-size: 1em;
+  width: 12.5em;
+  max-height: 2.375em;
+  min-width: 12.5em;
+}
+
+.logo svg {
+  width: 12.5em;
+  height: 2.375em;
+  display: block;
+  margin: 0 auto;
+}
+
+ /* The dropdown container */
+.dropdown {
+  overflow: hidden;
+  padding-inline: 40px;
+  margin-inline: 10px;
+  min-inline-size: 130px;
+}
```

Comment:
I'm not sure the dropdown quick-select adds any value over just using the sidebar for the megabook.

It will probably be useful when we have other books, though.